content/examples/databricks-cluster-remote.Rmd

```diff
@@ -0,0 +1,148 @@
+  operations.
+- Databricks Connect does not support most utilities in Databricks Utilities.
+  However, `dbutils.fs` and `dbutils.secrets` are supported.
+
+## Requirements
+
+- RStudio Server Pro
+- Java 8 installed on the server with RStudio Server Pro
+- Active Databricks cluster with a runtime version 5.5 and above
+
+## Install Python
+
+The Databricks Connect client is provided as a Python library.
+
+Follow the steps to install Python on the server with RStudio Server Pro:
+
+https://docs.rstudio.com/resources/install-python/
+
+Note that you can either install Python for all users in a global location or
+for an individual user in your home directory.
+
+## Install Databricks Connect
+
+Run the following command to install Databricks Connect on the server with
+RStudio Server Pro:
+
+```
+pip install -U databricks-connect==6.3.*  # or a different version to match your Databricks cluster
+```
+
+Note that you can either install this library for all users in a global Python
+environment or for an individual user in their Python environment (e.g., using
+the `pip --user` option or installing into a conda environment or virtual
+environment).
+
+## Configure Databricks Connect
+
+To configure the Databricks Connect client, you can run the following command in
+a terminal when logged in as a user in RStudio Server Pro:
+
+```
+databricks-connect configure
+```
+
+In the prompts that follow, enter the following information:
+
+Item | Description | Example Value
+---- | ----------- | -------------
+Databricks Host | Base address of your Databricks console URL | `https://dbc-01234567-89ab.cloud.databricks.com`
+Databricks Token | User token generated from the Databricks Console under your "User Settings" | `dapi24g06bdd96f2700b09dd336d5444c1yz`
+Cluster ID | Cluster ID in the Databricks console under Advanced Options > Tags > `ClusterId` | `0308-033548-colt989`
+Org ID | Found in the `?o=orgId` portion of your Databricks Console URL | `8498623428173033`
+Port | The port that Databricks Connect connects to | `15001`
+
+After you've completed the configuration process for Databricks Connect, you can
+run the following command in a terminal to test the connectivity of Databricks
+Connect to your Databricks cluster:
+
+```
+databricks-connect test
+```
+
+## Install `sparklyr`
+
+Note: The functionality to use `sparklyr` with Databricks Connect is under
+active development and is currently only available in the latest development
+version.
+
+To use `sparklyr` with Databricks Connect, you can install the development
+version of `sparklyr` by running the following R code:
+
+```
+devtools::install_github("sparklyr/sparklyr")
+```
+
+## Use `sparklyr`
+
+To work with a remote Databricks cluster, you need to have a local installation
+of Spark that matches the version of Spark on the Databricks Cluster.
+
+You can install Spark by running the following R code:
+
+```
+library(sparklyr)
+sparklyr::spark_install()
```

Comment:
It would be worth noting somewhere in here that you can specify the version of Spark you want in `sparklyr::spark_install()`

Reply:
Fixed in c8263f4.